Suko/API/SUKAPIManager.m

```diff
@@ -55,49 +55,49 @@ - (void)fetchSpecificAnimeByID:(NSNumber *) malID completion:(void(^)(SUKAnime*
     }];
 }
 
-- (void)fetchGenreAnime:(NSString *) genre completion:(void(^)(NSArray *arrofAnimeObjs, NSError *error))completion {
+- (void)fetchGenreAnime:(NSString *) genre completion:(void(^)(NSArray<SUKAnime*> *arrofAnimeObjs, NSError *error))completion {
```

Comment:
Maybe can name `arrofAnimeObjs` something different, so as to not get confused with `arrOfAnimeObjs` lol